mmv1/products/cloudrun/api.yaml

```diff
@@ -515,6 +518,34 @@ objects:
                     references will never be expanded, regardless of whether the variable
                     exists or not.
                     Defaults to "".
+                - !ruby/object:Api::Type::NestedObject
+                  name: valueFrom
+                  min_version: beta
+                  description: |-
+                    Source for the environment variable's value. Only supports secret_key_ref.
+                  properties:
+                    - !ruby/object:Api::Type::NestedObject
+                      name: secretKeyRef
```

Comment:
This should be marked `required: true` I think.

Required in api.yaml means that if the parent resource is specified, this one must be too. So if a user adds the `value_from` object, they must specify `secret_key_ref`.

Lets add `required: true` anywhere it makes sense to require a field. From my reading it should be most of these added fields?

Reply:
Done.